src/core/Akka.TestKit/TestKitBase_Expect.cs

```diff
@@ -368,7 +366,7 @@ private void AssertPredicateIsTrueForMessage<T>(Predicate<T> isMessage, T m, str
             CancellationToken cancellationToken)
         {
             return InternalExpectMsgAsync(timeout, msgAssert, senderAssert, hint, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.